src/Tests/Behat/Page/Admin/CustomBlock/ShowPage.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Lakion\SyliusCmsBundle\Tests\Behat\Page\Admin\CustomBlock;
+
+use Sylius\Behat\Page\SymfonyPage;
+
+class ShowPage extends SymfonyPage implements ShowPageInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getBlockContents()
+    {
+        return $this->getElement('block')->getHtml();
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function getDefinedElements()
+    {
+        return array_merge(parent::getDefinedElements(), [
```

Comment:
We can say something like that about every array or argument block. Why don't we put everything in multiple lines?